src/main/java/Duke.java

```diff
@@ -1,10 +1,331 @@
+            System.out.println("Added a Todo Task: " + getQueryDescription(query));
+            System.out.println("Total unchecked items in your list: " + getNumOfUncompletedTasks(tasks));
+            break;
+        case EVENT_TASK:
+            try {
+                if (getDate(query).equals("")) {
+                    throw new InvalidInputsException.InvalidDateFormatting("Wrong format of date "
+                            + "has been entered");
+                }
+                tasks.add(new Event(getQueryDescription(query), getDate(query)));
+                System.out.println("Added an Event Task: " + getQueryDescription(query));
+                System.out.println("Total unchecked items in your list: " + getNumOfUncompletedTasks(tasks));
+            } catch (InvalidInputsException.InvalidDateFormatting exception) {
+                exception.printStackTrace();
+                System.out.println("Did you forget to add in the '/by' again?");
+            }
+            break;
+        case DEADLINE_TASK:
+            try {
+                if (getDate(query).equals("")) {
+                    throw new InvalidInputsException.InvalidDateFormatting("Wrong format of date "
+                            + "has been entered");
+                }
+                tasks.add(new Deadline(getQueryDescription(query), getDate(query)));
+                System.out.println("Added a Deadline Task: " + getQueryDescription(query));
+                System.out.println("Total unchecked items in your list: " + getNumOfUncompletedTasks(tasks));
+            } catch (InvalidInputsException.InvalidDateFormatting exception) {
+                exception.printStackTrace();
+                System.out.println("Did you forget to add in the '/by' again?");
+            }
+            break;
+        case LIST_ITEMS:
+            printList(tasks);
+            break;
+        case DELETE_KEYWORD:
+            try {
+                int taskNumber = getTaskNum(query) - 1;
+                Task referencedTask = tasks.get(taskNumber);
+                tasks.remove(taskNumber);
+                printDeletedMessage(referencedTask);
+                System.out.println("Total unchecked items in your list: " + getNumOfUncompletedTasks(tasks));
+            } catch (IndexOutOfBoundsException exception) {
+                System.out.println("There is no such task number...");
+            }
+            break;
+        case NO_KEYWORD:
+            try {
+                throw new InvalidInputsException.MissingKeyword("You have to input <todo>, <deadline> or"
+                        + " <event> first!");
+            } catch (InvalidInputsException.MissingKeyword exception) {
+                exception.printStackTrace();
+                System.out.println("Invalid keyword!!!");
+            }
+            break;
+        case GOODBYE_KEYWORD:
+            printGoodbyeMessage();
+            break;
+        }
+    }
+
```

Comment:
This seems to be a very long method (>30 lines). You may consider extracting out the logic in each switch case into separate methods.